flink-connectors/flink-connector-hive/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,34 @@
+flink-connector-hive
```

Comment:
the transitive hive dependencies unfortunately have to be listed as well, as I pointed out in https://github.com/apache/flink/pull/8205#pullrequestreview-230102129 (at the bottom).

Reply:
I've changed hive-exec to be a test dependency now to not block us. I thought it is a not-so-hard problem to solve and include it when setting up flink-connector-hive but seems I was wrong. So let's leave it as test dependency now and change it to compile dependency whenever necessary later.